packages/block-library/src/post-comments/edit.js

```diff
@@ -213,6 +213,8 @@ export default function PostCommentsEdit( {
 										<span className="required">*</span>
 									</label>
 									<textarea
+										// eslint-disable-next-line no-restricted-syntax
+										id="comment"
```

Comment:
Why disable the eslint rule, rather than use its `withInstanceId` suggestion?

Reply:
I was about to say the same. 😄

@luisherranz take a look at this: https://github.com/WordPress/gutenberg/blob/086b77ed409a70a6c6a6e74dee704851eff812f2/packages/compose/src/higher-order/with-instance-id/README.md

Reply:
Or the [`useInstanceId`](https://github.com/WordPress/gutenberg/blob/trunk/packages/compose/src/hooks/use-instance-id/README.md) hook, rather than a HOC. 👀

Reply:
Oh, I think it might be because I've done the same for the Post Comments Form block in https://github.com/WordPress/gutenberg/pull/40368. I ll update my PR as well 🙂

Reply:
Because `comment` is the WordPress standard, and what it outputs in the frontend, so maybe some themes are using it. But I guess they shouldn't, right? And I had removed it previously anyway 😅

I'll update the PR.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/40527/commits/2ad190fac4e0ab0bd663487e2d97548d5c31ff8b.